src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -1305,30 +1305,36 @@ private void UpdateImage(int key, PropertyValue value)
                 if (_resourceUrl != null)
                 {
                     Size2D imageSize = ImageLoading.GetOriginalImageSize(_resourceUrl, true);
-
-                    int adjustedDesiredWidth, adjustedDesiredHeight;
-                    float aspectOfDesiredSize = (float)_desired_height / (float)_desired_width;
-                    float aspectOfImageSize = (float)imageSize.Height / (float)imageSize.Width;
-                    if (aspectOfImageSize > aspectOfDesiredSize)
+                    if( imageSize.Height > 0 && imageSize.Width > 0 )
```

Comment:
How about to check here to `_desired_width > 0 && _desired_height > 0`  ?

`_desired_width = 0` also have same potential of error